rgw/v2/tests/s3cmd/test_s3cmd.py

```diff
@@ -51,44 +54,95 @@ def test_exec(config):
     io_info_initialize = IOInfoInitialize()
     basic_io_structure = BasicIOInfoStructure()
     io_info_initialize.initialize(basic_io_structure.initial())
+    write_bucket_io_info = BucketIoInfo()
     umgmt = UserMgmt()
     ceph_conf = CephConfOp()
     rgw_service = RGWService()
     # preparing data
-    user_name = resource_op.create_users(no_of_users_to_create=1)[0]["user_id"]
-    tenant = "tenant"
-    tenant_user_info = umgmt.create_tenant_user(
-        tenant_name=tenant, user_id=user_name, displayname=user_name
-    )
-    user_info = umgmt.create_subuser(tenant_name=tenant, user_id=user_name)
-
     hostname = socket.gethostname()
     ip = socket.gethostbyname(hostname)
     port = utils.get_radosgw_port_no()
 
     ip_and_port = f"{ip}:{port}"
-    s3_auth.do_auth(tenant_user_info, ip_and_port)
 
-    bucket_name = utils.gen_bucket_name_from_userid(user_name, rand_no=0)
+    # Verifying CEPH-83574806
+    if config.delete_marker_check:
+        log.info(f"verification of testcase CEPH-83574806")
```

Comment:
@ckulal can we mention the scenario instead of Polarion Id

Reply:
Done